src/components/pages/doc/breadcrumbs/breadcrumbs.jsx

```diff
@@ -5,19 +5,19 @@ import Link from 'components/shared/link';
 import { DOCS_BASE_PATH } from 'constants/docs';
 
 const Breadcrumbs = ({ breadcrumbs }) => (
-  <div className="mb-4 flex space-x-1 text-sm leading-none text-gray-new-40 dark:text-gray-new-80 lg:hidden">
+  <div className="mb-4 flex space-x-2 text-sm leading-none text-gray-new-40 dark:text-gray-new-60 lg:hidden">
     {breadcrumbs.map(({ title, slug }, index) => (
       <Fragment key={index}>
         {index > 0 && <span>/</span>}
         {slug ? (
           <Link
-            className="transition-colors duration-200 hover:text-black dark:hover:text-gray-new-94"
+            className="transition-colors duration-200 hover:text-black dark:hover:text-white"
             to={DOCS_BASE_PATH + slug}
           >
             {title}
           </Link>
         ) : (
-          <span>{title}</span>
+          <span className="text-black dark:text-white">{title}</span>
```

Comment:
text-black-new?

Reply:
@vannyle we hadn't design for the light theme, so I just copied this color from the previous version. This color consists the body color and other elements on the Docs pages, so I guess it's ok to leave it just `text-black`